crm_deduplicate_acl/wizards/partner_merge_view.xml

```diff
@@ -3,29 +3,30 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
 
 <openerp>
-<data>
+    <data>
 
-<record id="base_partner_merge_automatic_wizard_form" model="ir.ui.view">
-    <field name="name">Restrict automatic merge access</field>
-    <field name="model">base.partner.merge.automatic.wizard</field>
-    <field
-        name="inherit_id"
-        ref="crm.base_partner_merge_automatic_wizard_form"/>
-    <field name="arch" type="xml">
-        <xpath expr="//button[@name='automatic_process_cb']"
-               position="attributes">
-            <attribute name="groups">
-                crm_deduplicate_acl.group_automatically
-            </attribute>
-        </xpath>
-        <xpath expr="//button[@name='update_all_process_cb']"
-               position="attributes">
-            <attribute name="groups">
-                crm_deduplicate_acl.group_automatically
-            </attribute>
-        </xpath>
-    </field>
-</record>
+        <record id="base_partner_merge_automatic_wizard_form" model="ir.ui.view">
```

Comment:
At least in my local computer if I do not change that, the application fails to recognize the groups. See how I added the permissions to demo user:
![image](https://cloud.githubusercontent.com/assets/7683926/16808824/5588966e-491e-11e6-8b24-4175ed040e7e.png)

But then when I open the wizard for deduplication, the automatic merge buttons are not visible:
![image](https://cloud.githubusercontent.com/assets/7683926/16808803/4013d208-491e-11e6-8ffd-52dc55974adc.png)

When I re-format the XML, they do show up correctly.